ops/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Arguments for controlling build details
 ###########################################################################################
 # Version of Triton to use
-ARG TRITON_VERSION=22.05
+ARG TRITON_VERSION=22.06
```

Comment:
I think this change is missing in build.sh: https://github.com/triton-inference-server/fil_backend/blob/2e3415ad4ce725f89f183f990bba16db6c8c177c/build.sh#L195

Reply:
Good catch! Fixed.